addons/mass_mailing/models/mass_mailing.py

```diff
@@ -517,21 +517,20 @@ def _onchange_mass_mailing_campaign_id(self):
             self.update(dic)
 
     @api.onchange('mailing_model_id', 'contact_list_ids')
-    def _onchange_model_and_list(self):
-        str_tuples = ""
+    def _onchange_model_and_list(self, mailing_domain=None):
```

Comment:
Why did you add a mailing_domain parameter ? Especially for an onchange method ?